src/mse_lib_sgx/cli.py

```diff
@@ -33,20 +38,36 @@ def parse_args() -> argparse.Namespace:
 
     parser.add_argument(
         "--host",
-        required=True,
         type=str,
-        help="hostname of the configuration server, "
-        "also the hostname of the app server if `--self-signed`",
+        default="0.0.0.0",
+        help="hostname of the server",
+    )
+    parser.add_argument("--port", type=int, default=443, help="port of the server")
+    parser.add_argument(
+        "--subject",
+        type=str,
+        help="Subject as RFC 4514 string for the RA-TLS certificate",
+    )
+    parser.add_argument(
+        "--san", type=str, help="Subject Alternative Name in the RA-TLS certificate"
     )
-    parser.add_argument("--port", required=True, type=int, help="port of the server")
     parser.add_argument(
         "--app-dir",
         required=True,
         type=Path,
         help="path of the python web application",
     )
     parser.add_argument(
-        "--uuid", required=True, type=str, help="unique application UUID"
+        "--id",
+        required=True,
+        type=uuid.UUID,
+        help="identifier of the application as UUID in RFC 4122",
+    )
+    parser.add_argument(
+        "--plaincode", action="store_true", help="unencrypted python web application"
```

Comment:
After further thoughts, i think we shouldn't pass this arg. We don't really need it : guess it during the configuration. 
And if you pass it here, we also need to pass it when we verify the mr enclave but it's completely useless because we do not decrypt the code when computing the mr enclave